docs/blog/2020-07-28-introducing-gatsby-i18n-theme/index.md

```diff
@@ -128,7 +128,7 @@ Time will tell what parts can be ported over to Gatsby’s codebase and what nee
 
 ### Why a theme and not a plugin?
 
-Composability is a major consideration for “Themes” and it should probably be more clear that you should work on top of it rather than it being a one-fits/one-click solution. We also might need to make use of gatsby-config.js which can only be used in a theme package.
+Composability is a major consideration for “Themes” and it should probably be more clear that you should work on top of it rather than it being a one-fits/one-click solution. We also might need to make use of `gatsby-config.js`which can only be used in a theme package.
```

Comment:
```suggestion
Composability is a major consideration for “Themes” and it should probably be more clear that you should work on top of it rather than it being a one-fits/one-click solution. We also might need to make use of `gatsby-config.js` which can only be used in a theme package.
```